pkg/build/controller/common/util.go

```diff
@@ -95,6 +95,7 @@ func HandleBuildPruning(buildConfigName string, namespace string, buildLister bu
 		sort.Sort(ByCreationTimestamp(successfulBuilds))
 
 		successfulBuildsHistoryLimit := int(*buildConfig.Spec.SuccessfulBuildsHistoryLimit)
+		glog.V(0).Infof("Current builds: %v, SuccessfulBuildsHistoryLimit: %v", len(successfulBuilds), successfulBuildsHistoryLimit)
```

Comment:
these still needs to be lower level, probably level 4 (though i realize that means you won't get this output during normal test runs).